### PR TITLE
Remove numbering and adjust other formatting issues

### PR DESCRIPTION
**Pull request by johannahemminger**
This doc is loooong

### CODE REVIEW

File: content/en/docs/appstore/modules/business-events.md [hunk 1]
```diff
@@ -152,19 +152,15 @@ The business event service document is open in Studio Pro:
 
 In the [next section](#add-be-definitions), you will define the information included in your events, as well as what the service will implement.
 
-##### 4.2.1.1 Defining Business Events {#add-be-definitions}
+##### 4.2.1.1 Define Business Events {#add-be-definitions}
```
Comment (Adam-Dupaski) [inline content/en/docs/appstore/modules/business-events.md]: @johannahemminger why change this to an infinitive when our standard is to use gerunds and it appears the rest of the doc uses gerunds?

Comment (Adam-Dupaski) [reply content/en/docs/appstore/modules/business-events.md]: Also, `1.2 Pre-Requisites` should be `1.2 Prerequisites` above

Comment (johannahemminger) [reply content/en/docs/appstore/modules/business-events.md]: I just made a few quick changes to create the PR and haven't gone through it all yet.

Comment (johannahemminger) [reply content/en/docs/appstore/modules/business-events.md]: I'm thinking  of completely renaming those sections... @Adam-Dupaski

**Pull request merged**